paint/support/src/dispatch.rs

```diff
@@ -23,11 +23,11 @@ pub use paint_metadata::{
 	FunctionMetadata, DecodeDifferent, DecodeDifferentArray, FunctionArgumentMetadata,
 	ModuleConstantMetadata, DefaultByte, DefaultByteGetter, ModuleErrorMetadata, ErrorMetadata
 };
+pub use crate::weights::{
```

Comment:
Is it really worth to re-export? This stuff is in the same crate, I don't see a point - it will just introduce ambiguity.

Reply:
AFAIK: This stuff, similar to other re-exports of the module are needed to be accessed from within the macro. I learned this by example. @bkchr can probably either explain more or correct me.